.travis.yml

```diff
@@ -5,25 +5,52 @@ cache:
     - $HOME/.composer/cache
 
 php:
-    - 5.5
-    - 5.6
-    - 7.0
+    - "5.5"
+    - "5.6"
+    - "7.0"
+    - "7.1"
+    - nightly
     - hhvm
 
 matrix:
     fast_finish: true
     include:
-        - php: 7.0
-          env: SYMFONY_VERSION=2.8.*
-        - php: 7.0
-          env: SYMFONY_VERSION=3.0.*
-        - php: 7.0
+        - php: "5.5"
+          env:
+              - "SYMFONY_VERSION=~2.8.0"
+              - "COMPOSER_FLAGS=--prefer-lowest"
+        - php: "7.0"
+          env: "SYMFONY_VERSION=~3.0.0"
+        - php: "7.0"
+          env: "SYMFONY_VERSION=~3.1.0"
+        - php: "7.0"
+          env: "SYMFONY_VERSION=~3.2.0@dev"
+    allow_failures:
+        - php: nightly
```

Comment:
you got a choice:

- want to keep track of wether or not your library is compatible with the next version, so you include it but allow it to fail (as it's not stable)
- you don't give a damn and not include it at all

You're call @dunglas, I don't feel strongly about this tbh.